internal/cli/daemon/client.go

```diff
@@ -55,13 +57,60 @@ func (c *octaviusClient) startOctaviusClient(grpcClient client.Client) error {
 	return nil
 }
 
+func validateImageName(imageName string) (bool, error) {
```

Comment:
I would suggest that we rename this to validateImageNameDocker to signify that in this function we validate for the image to be in docker registry. Later if we add a private registry we can just add a different local function and plug that where needed.